e2e/tests/stablex_test.rs

```diff
@@ -81,6 +82,10 @@ fn test_with_ganache() {
     // wait for solver to submit solution
     wait_for_condition(
         || {
+            // NOTE: Make sure that we periodically update the EVM time, as no
+            //   new blocks are being mined to update the EVM timestamp.
+            update_evm_time(&web3);
```

Comment:
I think we should colocate this with the thread::sleep statement in `wait_for_condition`. Note that we also have `wait_for` in the common.rs which we might want to remove now since it doesn't seem to work with the new system time approach.

Reply:
The `wait_for` is still helpful for speeding up being able to withdraw the balance.

Reply:
Makes perfect sense!